src/middlewared/middlewared/plugins/system.py

```diff
@@ -960,8 +973,22 @@ def _get_index(country_columns, column):
         `ui_address` and `ui_v6address` are a list of valid ipv4/ipv6 addresses respectively which the system will
         listen on.
 
-        When `syslogserver` is defined, `sysloglevel` makes sure that logs matching the specified level are sent.
+        `syslogserver` and `sysloglevel` are deprecated fields as of 11.3
+        and will be permanently moved to SystemAdvancedService for 12.0
```

Comment:
Please replace SystemAdvancedService with system.advanced.update.
API readers do not know what SystemAdvancedService is.